### PR TITLE
Add Abby Abbie to Shanghai

### PR DESCRIPTION
**Pull request by shijieyao**

### CODE REVIEW

**Comment by xavierchow** [comment]
@shijieyao thank you for your PR! Could you please update the README file to bump up the number for Shanghai as well?


image

**Comment by shijieyao** [comment]
@xavierchow  Done :D

File: shanghai.geojson [hunk 1]
```diff
@@ -1423,6 +1423,25 @@
...
+        "评论（@shijieyao）": "店面宽敞，有户外座，有多人座，附近是古北富人区，感觉顾客素质都较高，电源充足；双杯澳白是招牌，可选浅焙花香或深焙埃塞 SOE。",
+        "厕所": "没注意",
+        "marker-symbol": "cafe",
+        "marker-color": "#50C240"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          121.405,
+          31.200,
```
Comment (xavierchow) [inline shanghai.geojson:1442]:
```suggestion
          31.200
```

**Review by xavierchow — CHANGES_REQUESTED** [review]

**Review by xavierchow — APPROVED** [review]

**Comment by xavierchow** [comment]
merging, thank you!

**Pull request merged**